src/adapters/WstEthOracle.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.21;
+
+import {IStETH} from "../interfaces/IStETH.sol";
+import {AggregatorV3Interface} from "../interfaces/AggregatorV3Interface.sol";
+
+/// @title WstEthOracle
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice wstETH/ETH exchange rate price feed.
+/// @dev This contract should only be used as price feed for `ChainlinkOracle`.
+contract WstEthOracle is AggregatorV3Interface {
+    uint8 public constant decimals = uint8(18);
+    string public constant description = "wstETH/ETH exchange rate price";
+    uint256 public constant version = 1;
+
+    IStETH public immutable ST_ETH;
+
+    constructor(address stEth) {
+        require(stEth != address(0), "WstEthOracle: ZERO_ADDRESS");
```

Comment:
```suggestion
        require(stEth != address(0), "zero address");
```

specifying the context to me is overkill because the call context is well defined and can be traced in case of error

Considering `ErrorsLib` already exists, it seems better to have it there (without having to distinguish between `ChainlinkOracle` and adapters errors)